<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GBClientClass</class>
 <widget class="QWidget" name="GBClientClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1013</width>
    <height>605</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>GBClient</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,2">
   <item>
    <widget class="QWidget" name="ww" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QWidget" name="sip_info_area" native="true">
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QWidget" name="horizontalWidget" native="true">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>10</height>
               </size>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_12" stretch="1,3">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>SipIp:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="sip_ip_line"/>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>SipId:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="sip_id_line"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,3">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>SipPort:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="sip_port_line"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,3">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Username:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="username_line"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,3">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Password:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="password_line"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,3">
              <item>
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Proxy:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="proxy_combo">
                <item>
                 <property name="text">
                  <string>UDP</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>TCP</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="listen_btn">
             <property name="text">
              <string>Listen</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="stop_listen_btn">
             <property name="text">
              <string>stop</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QTreeWidget" name="device_tree">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <attribute name="headerVisible">
            <bool>false</bool>
           </attribute>
           <column>
            <property name="text">
             <string notr="true">1</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="1,6,1">
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>1</width>
               <height>18</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,2,1">
               <item>
                <widget class="QPushButton" name="query_btn">
                 <property name="text">
                  <string>Query</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QWidget" name="widget_3" native="true">
                 <property name="minimumSize">
                  <size>
                   <width>100</width>
                   <height>0</height>
                  </size>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_9">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <property name="leftMargin">
                   <number>1</number>
                  </property>
                  <property name="topMargin">
                   <number>1</number>
                  </property>
                  <property name="rightMargin">
                   <number>1</number>
                  </property>
                  <property name="bottomMargin">
                   <number>1</number>
                  </property>
                  <item>
                   <widget class="QComboBox" name="stream_combo">
                    <item>
                     <property name="text">
                      <string>UDP</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>TCP</string>
                     </property>
                    </item>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="play_btn">
                    <property name="text">
                     <string>Play</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="stop_btn">
                 <property name="text">
                  <string>Stop</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>1</width>
               <height>18</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5" stretch="3,1">
     <item>
      <widget class="QWidget" name="video_view" native="true">
       <layout class="QGridLayout" name="video_layout"/>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="message_browser"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="gbclient.qrc"/>
 </resources>
 <connections/>
</ui>
